packages/editor-ui/src/stores/users.store.test.ts

```diff
@@ -58,4 +63,29 @@ describe('users.store', () => {
 			expect(identify).toHaveBeenCalledWith('test-instance-id', mockUser.id);
 		});
 	});
+
+	describe('inviteUsers', () => {
+		it('should add pending user to the store', async () => {
+			const usersStore = useUsersStore();
+
+			inviteUsers.mockResolvedValueOnce([
+				{
+					user: { role: 'global:member', email: 'test@n8n.io', emailSent: true },
```

Comment:
```suggestion
					user: { id: 'random-id', email: 'test@n8n.io', emailSent: true },
```

The API does not return the role, but the id.